Corale.Colore/Razer/Result.cs

```diff
@@ -347,10 +347,10 @@ public override string ToString()
             foreach (var fieldInfo in fieldsInfo.Where(fi => fi.FieldType == typeof(Result)))
             {
                 var value = fieldInfo.GetValue(null);
-                var attr = fieldInfo.GetCustomAttribute<DescriptionAttribute>(false);
+                var attr = fieldInfo.GetCustomAttributes(typeof(DescriptionAttribute),false)[0];
```

Comment:
`GetCustomAttributes(Type, bool)` can return a 0-length array, so this should probably be `var attrs` instead, retrieving the whole array and then checking for `attr.Length > 0 && attr[0] is DescriptionAttribute)`.

Actually we wouldn't need to check the Type at all would we? If there is at least 1 element in the array, that means the DescriptionAttribute was found and it's restricted to 1 attribute per member anyway.